### PR TITLE
Remove Printer From Website

### PR DESCRIPTION
**Pull request by justindhillon**
The cube does not have a printer anymore, so I removed all the printer parts of the website along with all the links to the printer.

### CODE REVIEW

File: content/posts/2007-02-13-an-evening-at-telus-science-world-for-women-in-science-engineering-and-technology.md
Comment (kewbish) [inline content/posts/2007-02-13-an-evening-at-telus-science-world-for-women-in-science-engineering-and-technology.md:1]: This is its own file, please leave it as-is!

**Review by kewbish — APPROVED** [review]
Please address the below, and I'll merge!

File: content/posts/2007-02-13-an-evening-at-telus-science-world-for-women-in-science-engineering-and-technology.md
Comment (justindhillon) [reply content/posts/2007-02-13-an-evening-at-telus-science-world-for-women-in-science-engineering-and-technology.md:1]: Sorry about that, it is fixed now.

**Comment by kewbish** [comment]
Could you address the merge conflicts as I did on the other PR?

**Comment by justindhillon** [comment]
I couldn't understand git rebase, so I just made the commits again.

**Pull request merged**